eno-core/src/main/java/fr/insee/eno/core/annotations/InAnnotationValues.java

```diff
@@ -0,0 +1,10 @@
+package fr.insee.eno.core.annotations;
+
+/**
+ * Inheritance of annotations is not possible in Java.
+ * The record is designed to hold the information of an input annotation, and to be used in a polymorphic way.
+ */
+public record InAnnotationValues(
```

Comment:
Faute d'héritage entre les annotations, on peut utiliser la composition :on pourrait avoir une annotation générique (par exemple @InMapping avec ces trois attributs) et @Pogues et @DDI pourraient dépendre de cette annotation ainsi : ` @Pogues(in=@InMapping("getId()"))`. Les méthodes génériques sur l'ensemble des annotations de mapping pourraient utiliser des objets InMapping

Reply:
Je vois l'idée, mais je préfère rester avec mon record, certes ça implique de recopier les propriétés des annotations de mapping, mais ça me paraît plus important que les annotations qui apparaissent un peu partout sur le modèle soient le moins verbeuse possible à l'usage.